### PR TITLE
Fall back to AQHI forecast

### PR DESCRIPTION
**Pull request by michaeldavie**
Uses the next hourly forecast value if a current observation is not available.

### CODE REVIEW

File: homeassistant/components/environment_canada/sensor.py [hunk 1]
```diff
@@ -215,7 +215,9 @@ class ECSensorEntityDescription(
     device_class=SensorDeviceClass.AQI,
     native_unit_of_measurement="AQI",
     state_class=SensorStateClass.MEASUREMENT,
-    value_fn=lambda data: data.current,
+    value_fn=lambda data: data.current
```
Comment (gwww) [inline homeassistant/components/environment_canada/sensor.py:218]: This won't be allowed as written. `lambda`s of more than one line are frowned upon.

**Review by gwww — COMMENTED** [review]
It seems I should put the PR into draft as this change is to what the PR is already changing. Does that seem right?

**Comment by michaeldavie** [comment]
Maybe? With these tweaks to the the library and the sensor, I don't think we need to disable the AQHI by default.

Is it possible to use a named function for the fallback instead of the lambda?

**Comment by gwww** [comment]
I've change the PR to draft until we get this done. It shouldn't take long, just don't what is there accepted only to change it again.

**Pull request closed without merging**